android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMHardwareKeyboardInterpreter.java

```diff
@@ -114,18 +116,70 @@ public KMHardwareKeyboardInterpreter(Context context, KMManager.KeyboardType key
 
   @Override
   public boolean onKeyDown(int keyCode, KeyEvent event) {
+    // Note: Keep this table in sync with kmwosk.js osk.modifierCodes
```

Comment:
This really should be in a separate class given it is definition information that we could use elsewhere as well.

Reply:
Moving to a separate `KMModiferCodes` class